CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# 1.1.13 (2022-04-27)
+## Edge Agent
+### Bug Fixes
+* Update Base Images to address gzip vulnerability [CVE-2022-1271](https://ubuntu.com/security/CVE-2022-1271)
+
+## Edge Hub
+### Bug Fixes
+* Update Base Images to address gzip vulnerability [CVE-2022-1271](https://ubuntu.com/security/CVE-2022-1271)
+
+## iotedged
+### Bug Fixes
+* Update Rust `regex` version to address vulnerability [5573eff](https://github.com/Azure/iotedge/commit/5573eff191310e5a226d924b40d1cea035b0a3fd)
+* Publish arm32v7 packages for Debian 9, 10, and 11 to packages.microsoft.com [3e9604b](https://github.com/Azure/iotedge/commit/3e9604b67b6320dba4aab22c4d81b4db67e0c974)
+
+## Diagnostics Module
+### Bug Fixes
+* Update Base Images to address gzip vulnerability [CVE-2022-1271](https://ubuntu.com/security/CVE-2022-1271)
+
+## Metrics Collector
```

Comment:
I think you can leave metrics collector out of this changelog, since it's a separate release with its own changelog.

Reply:
Ok, I removed that section.

Reply:
With that said though, we need to remember to do the release of that independently.